use-cases/predictive_maintenance/demo_helpers.py

```diff
@@ -55,16 +55,4 @@ def update_dw_s3uri(flow_file_name):
 
     with open(flow_file_name, "w") as f:
         json.dump(flow, f)
-
-
-dw_container_dict = {
-    "us-east-2": "415577184552.dkr.ecr.us-east-2.amazonaws.com/sagemaker-data-wrangler-container:1.0.1"
-}
```

Comment:
Good to get rid of this hard-coding, but how is the Data Wrangler image being fetched now? Is it using the SM SDK from within the notebook(s)?

Reply:
Yes, it is being fetched using sagemaker.image_uris.retrieve function now